monitor/hosts.go

```diff
@@ -114,8 +114,13 @@ func terminateHosts(ctx context.Context, hosts []host.Host, settings *evergreen.
 		work <- &hosts[i]
 	}
 	close(work)
-
-	for i := 0; i < 12; i++ {
+	workers := 24
+	if len(hosts) > workers {
```

Comment:
Is your goal here really to have as many workers as hosts, and to have excess workers if there are fewer than 24 hosts to terminate?